x-pack/plugins/osquery/package.json

```diff
@@ -7,10 +7,10 @@
   "scripts": {
     "cypress:burn": "yarn cypress:run --env burn=2 --headed",
     "cypress:changed-specs-only": "yarn cypress:run --changed-specs-only --env burn=2",
-    "cypress": "node ../security_solution/scripts/start_cypress_parallel --config-file ../osquery/cypress.config.ts --ftr-config-file ../../../x-pack/test/osquery_cypress/cli_config",
+    "cypress": "NODE_OPTIONS=--openssl-legacy-provider && node ../security_solution/scripts/start_cypress_parallel --config-file ../osquery/cypress.config.ts --ftr-config-file ../../../x-pack/test/osquery_cypress/cli_config",
     "cypress:open": "yarn cypress open",
     "cypress:run": "yarn cypress run",
-    "cypress:serverless": "node ../security_solution/scripts/start_cypress_parallel --config-file ../osquery/serverless_cypress.config.ts --ftr-config-file ../../../x-pack/test/osquery_cypress/serverless_cli_config",
+    "cypress:serverless": "NODE_OPTIONS=--openssl-legacy-provider && node ../security_solution/scripts/start_cypress_parallel --config-file ../osquery/serverless_cypress.config.ts --ftr-config-file ../../../x-pack/test/osquery_cypress/serverless_cli_config",
```

Comment:
I believe we don't need `&&` here

Reply:
good catch, fixed it here https://github.com/elastic/kibana/pull/164603/commits/0af487bafc3a923b23a566c2772d1b01a9f011ee